tests/baselines/reference/arrowFunctionContexts.js

```diff
@@ -116,7 +116,8 @@ var Base = (function () {
 var Derived = (function (_super) {
     __extends(Derived, _super);
     function Derived() {
-        return _super.call(this, function () { return _this; }) || this;
+        var _this = _super.call(this, function () { return _this; }) || this;

```

Comment:
This is intentional - it is an error for `this` to be used before a call to `super(...)` completes.

This means that it shouldn't be an error to *reference* `_this`, but it should be an error to evaluate `_this` in a callback before the `super(...)` call returns.